src/firefly/tests/test_voltmeters.py

```diff
@@ -151,14 +167,65 @@ def check_item(item):
         display.ui.auto_gain_button.click()
 
 
+async def test_shutters_checkbox_no_shutters(voltmeters_display, sim_registry):
+    display = voltmeters_display
+    combobox = display.ui.shutter_combobox
+    checkbox = display.ui.shutter_checkbox
+    checkbox.setChecked(True)
+    # Update the state of the UI with no shutters
+    await display.update_devices(sim_registry)
+    # Ensure checkbox has been disabled
+    assert not checkbox.isEnabled()
+    assert not checkbox.checkState()
+    assert not display.ui.shutter_checkbox.setChecked(True)
+    combobox_items = [combobox.itemText(idx) for idx in range(combobox.count())]
+    assert len(combobox_items) == 0
+
+
+async def test_shutters_checkbox_with_shutters(
+    voltmeters_display,
+    sim_registry,
+    shutters,
+):
+    display = voltmeters_display
+    checkbox = display.ui.shutter_checkbox
+    combobox = display.ui.shutter_combobox
+    # Update the state of the UI with no shutters
+    await display.update_devices(sim_registry)
+    # Ensure checkbox has been enabled
+    assert checkbox.isEnabled()
+    # Check that shutters were added to the combobox
+    combobox_items = [combobox.itemText(idx) for idx in range(combobox.count())]
+    assert "front_end_shutter" in combobox_items
```

Comment:
If we decided to remove front end shutter, the test should be modified too

Reply:
Done.